js/api/dispatch-v2/index.js

```diff
@@ -43,14 +40,42 @@ var wsServer = new WebSocketServer({
     },
 })
 
-let isClosing = false
+let isClosing = false,
+    couriersList = {}
+
+const channels = [
+  'task:unassign',
+  'task:assign',
+  'task:done',
+  'task:failed'
+]
+
+_.each(channels, (channel) => { sub.prefixedSubscribe(channel) })
+
+sub.on('subscribe', (channel, count) => {
+  if (count === channels.length) {
+    sub.on('message', function(channelWithPrefix, message) {
+      const parsedMessage = JSON.parse(message),
+            username = parsedMessage.user.username
+      _.each(channels, (channel) => {
+        if (sub.isChannel(channelWithPrefix, channel) && couriersList[username]) {
+          winston.debug(`Sending message ${message} to ${username}`)
+          couriersList[username].send(message)
```

Comment:
I don't understand how you distinguish between assign/unassign on the client..
Maybe it would be better to also send the channel name?

```
couriersList[username].send({
  type: channel,
  data: message
})
```

Reply:
Yes indeed I made this fix locally, forgot to push afterwards -_-

Reply:
Push to master :shipit: